metrics/details/init.lua

```diff
@@ -99,6 +99,21 @@ end
 
 global_metrics_registry = Registry.new()
 
+------------------------------- Global Labels --------------------------------
+global_labels = {}
+
+local function append_global_labels(label_pairs)
+    local extended_label_pairs = table.deepcopy(label_pairs)
+
+    for k, v in pairs(global_labels) do
```

Comment:
Let's verify "global_labels" in "set_global_labels" function.

I think we should merge only correct and checked values

Reply:
I did not find any explicit specification on labels. All three drivers imply we must be able to cast both key and value to strings, so I verify it now. (Correct `set_global_labels()` arguments are `nil`, `{}` or `{ string_like_key = string_like_value, ... }`.)